tools/ipapy2/ipapy2_isotope_patterns.xml

```diff
@@ -0,0 +1,96 @@
+<tool id="ipapy2_isotope_patterns" name="3. IPA isotope patterns" version="@TOOL_VERSION@+galaxy0" python_template_version="3.5" profile="21.05">
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">ipapy2</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+        python3 '${ipapy2_isotope_cli}'
+    ]]></command>
+
+<configfiles>
+<configfile name="ipapy2_isotope_cli">
+@init_logger@
+
+ipa_dataframe = pd.read_csv('${isotope_input}')
+#if $ionisation == "positive"
+ipa.map_isotope_patterns(ipa_dataframe, isoDiff=$isotope_diff, ppm=$ppm, ionisation=1, MinIsoRatio=$isotope_ratio)
+#else
+ipa.map_isotope_patterns(ipa_dataframe, isoDiff=$isotope_diff, ppm=$ppm, ionisation=-1, MinIsoRatio=$isotope_ratio)
+#end if
```

Comment:
Same about the ionization parameter - this if clause can be removed if the select is parameterized correctly.

Reply:
This was resolved in https://github.com/RECETOX/galaxytools/pull/474/commits/04166cde650d100dbc94873bd90959ee9c818b0a